tests/ui_tests/test_admin_page.py

```diff
@@ -382,16 +383,16 @@ def test_create_policy_on_admin_groups_page(self, app_fs):
             self.custom_policy.groups,
         )
         current_policies = policies_page.get_all_policies()
-        with allure.step('Check that there are 2 policies'):
-            assert len(current_policies) == 2, "There should be 2 policies on the page"
-            assert current_policies == ["default", self.custom_policy.name], "Created policy should be on the page"
+        with allure.step('Check that there is 1 policy'):
+            assert len(current_policies) == 1, "There should be 1 policy on the page"
+            assert current_policies == [self.custom_policy], "Created policy should be on the page"
```

Comment:
Will they be equal because they're both dataclasses? Because regular objects will have problems with that.
If dataclasses are hashable I suggest use here sets to avoid questions like mine in the future)

Reply:
works correctly